src/index.test.ts

```diff
@@ -135,146 +207,192 @@ describe("tts.websocket", () => {
     phonicWebSocket.close();
   });
 
-  test("send text and receive all audio chunks", async () => {
-    const phonic = new Phonic(apiKey, { baseUrl });
-    const { data, error } = await phonic.tts.websocket();
+  test("generate(short)-flush", async () => {
+    const text = "Hello, world!";
+    phonicWebSocket.generate({ text });
+    phonicWebSocket.flush();
 
-    if (error !== null) {
-      expect(error).toBeNull();
-      return;
-    }
+    const messages = await allMessagesReceived;
 
-    const { phonicWebSocket } = data;
+    expect(messages).toHaveLength(3);
```

Comment:
Checking the individual messages below should cover this, right?
If you end up getting e.g. only 2 messages here, this would fail but won't give you any details except that the number is wrong. I think it's better to delete this line and rely on the individual checks below.

Reply:
I also wanted to check that we didn't get any extra messages back. I could create a separate helper function that asserts we have at most N messages, and then rely on the individual checks to catch if we have fewer messages. Would you prefer that?

Reply:
I see, all good, let's leave it as is.